packages/analytics/analytics-chart/src/components/chart-types/StackedBarChart.vue

```diff
@@ -359,18 +360,22 @@ const chartWidth = computed(() => {
 })
 
 const chartHeight = computed(() => {
-  let chartHeight = parseInt(props.height, 10)
-  if (numLabels.value && isHorizontal.value) {
+  let value: number | string = DEFAULT_CHART_HEIGHT
```

Comment:
suggestion: can you add a return type to the computed function, and add some comments as to what is going on within this computed variable?